docs/src/model_setup/output_writers.md

```diff
@@ -163,38 +163,27 @@ To use this functionality, the `output_grid` must be passed explicitly when cons
 
 ```jldoctest
 using Oceananigans
+using Oceananigans.Fields: interpolate!
 
-grid = RectilinearGrid(size = (1, 1, 8), extent = (1,1,1));
-model = NonhydrostaticModel(; grid, closure = ScalarDiffusivity(ν=1e-2))
-
-set!(model, u=(x, y, z,) -> z)
-
-simulation = Simulation(model,
-                        Δt=0.5*maximum(zspacings(grid, Center())) / maximum(abs, model.velocities.u),
-                        stop_time=20)
-
-simulation.output_writers[:fullfields] = NetCDFOutputWriter(model, (; model.velocities.u),
-                                                            filename = "fullfields.nc",
-                                                            schedule = TimeInterval(5),
-                                                            overwrite_existing = true,)
+grid = RectilinearGrid(size=(1, 1, 8), extent=(1, 1, 1));
+model = NonhydrostaticModel(; grid)
+simulation = Simulation(model, Δt=1, stop_iteration=1)
 
-coarse_grid = RectilinearGrid(size = (grid.Nx, grid.Ny, grid.Nz÷2), extent = (grid.Lx, grid.Ly, grid.Lz))
+coarse_grid = RectilinearGrid(size=(grid.Nx, grid.Ny, grid.Nz÷2), extent=(grid.Lx, grid.Ly, grid.Lz))
 coarse_u = Field{Face, Center, Center}(coarse_grid)
 
-using Oceananigans.Fields: interpolate!
-update_coarse_u(simulation) = interpolate!(coarse_u, simulation.model.velocities.u)
-simulation.callbacks[:update_interp] = Callback(update_coarse_u)
+interpolate_u(model) = interpolate!(coarse_u, model.velocities.u)
+outputs = (; u = interpolate_u)
 
-simulation.output_writers[:coarsefields] = NetCDFOutputWriter(model, (; coarse_u,), coarse_grid;
-                                                              filename="coarsefields.nc",
-                                                              schedule=TimeInterval(5),
-                                                              overwrite_existing=true,)
+simulation.output_writers[:coarse_u] = NetCDFOutputWriter(model, outputs, coarse_grid;
+                                                          filename = "coarse_u.nc",
+                                                          schedule = IterationInterval(1))
```

Comment:
We don't need `Simulation` for this example right?

Reply:
hm... yes, but all other previous examples for the same docs is defining like this.

Reply:
Sometimes, we must leave the past behind for a better, brighter future.

Reply:
Agreed! But we also need to be consistent, right? Or not really for that case?

Reply:
What about this?

```julia
using Oceananigans
using Oceananigans.Fields: interpolate!

grid = RectilinearGrid(size=(1, 1, 8), extent=(1, 1, 1));
model = NonhydrostaticModel(; grid)

coarse_grid = RectilinearGrid(size=(grid.Nx, grid.Ny, grid.Nz÷2), extent=(grid.Lx, grid.Ly, grid.Lz))
coarse_u = Field{Face, Center, Center}(coarse_grid)

interpolate_u(model) = interpolate!(coarse_u, model.velocities.u)
outputs = (; u = interpolate_u)

output_writer = NetCDFOutputWriter(model, outputs, coarse_grid;
                                    filename = "coarse_u.nc",
                                    schedule = IterationInterval(1))
```

Reply:
> Agreed! But we also need to be consistent, right? Or not really for that case?

I think its more important to innovate, we can change the other examples if we come up with a better / more concise pattern.

Reply:
Yes, I think that there is a balance between conciseness and usability that we need to be at. I kind of see the value of having a simulation because this is inevitably most of the cases when you need to create a writer, but at the same time, we want the documentation to be simple and not being used as simply as copy and paste. What do you think about that?

Reply:
I simplified this example, but not the other examples as they explicitly mention the file saving.

Reply:
sounds good